grails-forge-core/src/main/java/org/grails/forge/feature/database/DatabaseDriverFeature.java

```diff
@@ -65,7 +65,16 @@ public String getCategory() {
 
     public abstract boolean embedded();
 
-    public abstract String getJdbcUrl();
+    @Deprecated()
```

Comment:
Should we add: `@Deprecated(since = "6.2.2", forRemoval = true)`?

Reply:
updated